notebooks/extracting_metadata_filters_from_a_user_query.ipynb

```diff
@@ -0,0 +1,878 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #47.            # this can be done with specific data structures and in a more sophisticated way
</pre><p>although this makes sense, we can maybe remove it since it's a cookbook - I don't know, you can decide :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I wanted to leave as it provides a nice improvement point :)